src/Disks/S3/DiskS3.cpp

```diff
@@ -320,6 +320,8 @@ void DiskS3::startup()
 {
     auto settings = current_settings.get();
 
+    settings->client->EnableRequestProcessing();
```

Comment:
How it change behavior, what does it actually do? AWS SDK doc says `Enables/ReEnables request processing`

Reply:
It's missed call when we do `DiskS3::shutdown()` and invoke `client->DisableRequestProcessing();`
it can lead to breaking requests without retrying after disk restart.